lib/main.dart

```diff
@@ -124,10 +124,22 @@ Future<void> main() async {
 
   // Colors
   final theme = ThemeData(
```

Comment:
I would move the Theme inside of the theme.dart and remove the theme from the main file.